drizzle/schema.ts

```diff
@@ -74,6 +74,7 @@ export const teams = pgTable("teams", {
 		.defaultNow()
 		.notNull()
 		.$onUpdate(() => new Date()),
+	isInternalTeam: boolean("is_internal_team").notNull().default(false),
```

Comment:
@satococoa 
If we use a boolean like `isInternalTeam`, there is a concern that adding more team types for other purposes in the future could lead to an increase in columns and a combinatorial explosion. 

From my past experience, dealing with such changes was quite challenging, so since this change involves modifying the database table, how about using an enum from the beginning instead of a boolean?

Reply:
> The term "combinatorial explosion" might be an exaggeration, but having to implement logic to handle combinations of multiple booleans could create a situation prone to bugs.

Yes, I think so.

> how about using an enum from the beginning instead of a boolean?

Agreed.

I'll make the changes following your advice.
Thank you!